docs/installation.rst

```diff
@@ -17,19 +17,19 @@ Create your pulp_ansible.yml playbook to use with the installer:
 
 .. code-block:: yaml
 
-   ---
-   - hosts: all
-     vars:
-       pulp_settings:
-         secret_key: << YOUR SECRET HERE >>
-         content_origin: "http://{{ ansible_fqdn }}"
-       pulp_default_admin_password: << YOUR PASSWORD HERE >>
-       pulp_install_plugins:
-         pulp-ansible: {}
-       roles:
-         - pulp.pulp_installer.pulp_all_services
-       environment:
-       DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+  ---
+  - hosts: all
+    vars:
+      pulp_settings:
+        secret_key: << YOUR SECRET HERE >>
+        content_origin: "http://{{ ansible_fqdn }}"
+      pulp_default_admin_password: << YOUR PASSWORD HERE >>
+      pulp_install_plugins:
+        pulp-ansible: {}
+    roles:
+      - role: pulp.pulp_installer.pulp_all_services
+        environment:
+          DJANGO_SETTINGS_MODULE: pulpcore.app.settings
```

Comment:
```suggestion
    environment:
      DJANGO_SETTINGS_MODULE: pulpcore.app.settings
```

I cannot find role level environment in the ansible docs...

Reply:
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_environment.html

Reply:
Right, thank you. Fixed in 66c9bd0b33976b3a4a4707962eae357bed027bf8